### PR TITLE
Hello world in Java

### PR DESCRIPTION
**Pull request by Istiakmorsalin**

### CODE REVIEW

File: HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* This is a simple Java hello world program.
+   FileName : "HelloWorld.java". */
+class HelloWorld {
+    // Your program begins with a call to main().
+    // Prints "Hello, World" to the terminal window.
+    public static void main(String args[]) {
+        System.out.println("Hello, World");
```
Comment (mirshahriar) [inline HelloWorld.java]: Hello, World!

**Review by mirshahriar — CHANGES_REQUESTED** [review]

File: HelloWorld.java (hunk 1 above)
Comment (nafSadh) [reply HelloWorld.java]: Yes, please update to print "'Hello, World!". Also, add move the source file within a reasonable package: e.g.: `com.example.hello`

Comment (Istiakmorsalin) [reply HelloWorld.java]: @nafSadh check now

File: HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.example.hellolang;
+/* This is a simple Java hello world program.
+   FileName : "HelloWorld.java". */
+class HelloWorld {
+    // Your program begins with a call to main().
+    // Prints "Hello, World" to the terminal window.
+    public static void main(String args[]) {
+        System.out.println("Hello, World!");
+    }
+}
```
Comment (nafSadh) [inline HelloWorld.java:10]: Please add a newline.

Comment (Istiakmorsalin) [reply HelloWorld.java:10]: `println` throws the cursor to the next line automatically. I do not understand why do we have to add a newline here.

**Review by mirshahriar — APPROVED** [review]
LGTM

File: HelloWorld.java (hunk 2 above)
Comment (nafSadh) [reply HelloWorld.java:10]: Java source file (and most other languages) usually ends with a newline. GitHub showed an warning.

**Pull request merged**